authentication.md

```diff
@@ -8,26 +8,27 @@ https://github.com/stormpath/stormpath-sdk-spec/blob/master/specifications/authe
 Those authenticators have an interface which is *not* HTTP-specific, the
 interfaces work with the necessary parameters for achieving authentication.
 
-As such, our framework integrations will be responsible for taking an incoming
-HTTP request, gathering the needed authentication parameters (from the body,
-from HTTP headers, etc) and delegating to the necessary authenticator.
+To provide extra convenience, we have defined Account Resolvers to help the application read the HTTP request and return authentication information to the application. In addition, the framework integrations MAY provide additional convenience methods to respond to the incoming request. 
 
-This work should be encapsulated into re-usable functions that can be exposed
-in the public API of the framework.
+**Account Resolvers**
 
-The default set of functions that we want to provide are:
+The framework integration will provide the application with some means of resolving authentication information. This means:
 
-* BasicRequestAuthenticator
-* OAuthBearerRequestAuthenticator
-* OAuthClientCredentialsRequestAuthenticator
-* OAuthPasswordRequestAuthenticator
-* OAuthRefreshTokenRequestAuthenticator
-* OAuthStormpathTokenRequestAuthenticator
+* Reading the request headers for authentication information
+* Resolving the access token or API key presented to a Stormpath account
+* Return this information to the application, along with the method of authentication.
 
-We are evaluating the possibility of "smart" request authenticators, that can
-take any request and delegate to the specific request authenticator.  For
-example, a "OAuthRequestAuthenticator", which will delegate to the proper
-OAuth request authenticator.
+This information will allow the application to make authorization decisions for the request. 
+
+**Unauthenticated Requests**
```

Comment:
This section can be clarified to describe the fact that this is something that happens after the developer has required authentication on their API, using something that we provide to require that authentication.